docs/advanced-usage/pytorch-classifier.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 4
+id: pytorch-classifier
+title: Integrating Your Custom PyTorch Image Classifier
+---
+
+In this guide, we will walk you through the process of leveraging the `TORCHSCRIPT_CLASSIFIER` node in Flojoy to integrate a custom image classification model. Specifically, we'll use a Colab notebook to train the `MobileNetV3` model on a cats and dogs dataset, export it as a `.torchscript` format, and then deploy it within the `TORCHSCRIPT_CLASSIFIER` node.
+
+### 1. Train and Export Your Model
+
+Start by accessing the provided Colab notebook which serves as an ideal foundation for training and exporting your model. This notebook is tailored for a cats vs. dogs dataset, but you can effortlessly modify it to suit different datasets with varying class numbers.
+
+- **Colab Notebook**: [Link](https://colab.research.google.com/drive/1HCj3i43tYt4CTLPHq4BqVC2HL0lAN2-X?usp=sharing)
+
+To use the notebook:
+
+1. Open the notebook link.
+2. Run all cells sequentially.
+3. Upon completion, you'll find the `model.torchscript` file ready for download in the left sidebar, as illustrated here:
+
+![image](../../static/img/advanced_tutorials/TORCHSCRIPT_TUTORIAL_SIDEBAR.png)
+
+### 2. Deploy the Model in `TORCHSCRIPT_CLASSIFIER`
+
+To utilize a trained model within the `TORCHSCRIPT_CLASSIFIER` node, ensure you have:
+
+- The exported `.torchscript` model file.
+- A `.csv` file mapping class names to class IDs.
+
+For instance, your `csv` file for a cats vs. dogs classifier should look like this:
+
+```
+class_name
+"cat"
+"dog"
+```
+
+Now, with these files at hand, you're all set to integrate your custom PyTorch Image Classifier into Flojoy using the `TORCHSCRIPT_CLASSIFIER` node.
+
+### 3. [Optional] Train the model on your own data
+
+If you'd like to train the model on your own data, your dataset needs to be compatible with the PyTorch `ImageFolder` dataset format. This essentially means that your images need to be structured in a directory hierarchy such as:
+
+```
+dataset/
+├── class_1
+│   ├── img1.jpg
+│   ├── img2.jpg
+│   ├── ...
+├── class_2
+│   ├── img1.jpg
+│   ├── img2.jpg
+│   ├── ...
+├── ...
+```
+
+For more information on the `ImageFolder` dataset format, refer to the PyTorch documentation [here](https://pytorch.org/vision/stable/generated/torchvision.datasets.ImageFolder.html#torchvision.datasets.ImageFolder).
+
+
+You will need to curate both (a) training and (b) test sets with the format described above, and compress them together in a zip file containing a `training_set` folder and a `test_set` folder.
```

Comment:
To me, we must explain how we evaluation of models is performed. For now, there is an ambiguity in the ways models are trained and evaluated. In the notebook, `val_loader`, the Data Loader for validation, use the `test_set` ImageFolder.

How about renaming `test_set` as `validation_set`?

```suggestion
You will need to create an zip archive consisting of two folders `training_set` and `validation_set`.

Both `training_set` and `validation_set` have to comply with the [`torchvision.datasets.ImageFolder`](https://pytorch.org/vision/stable/generated/torchvision.datasets.ImageFolder.html#torchvision.datasets.ImageFolder) format.
```

Reply:
Valid point, thanks for nuancing this